src/main/java/com/everis/boundary/CarResources.java

```diff
@@ -0,0 +1,88 @@
+package com.everis.boundary;
+
+import java.util.List;
+
+import javax.ejb.EJB;
+import javax.ws.rs.*;
+import javax.ws.rs.core.*;
+import javax.ws.rs.core.Response.*;
+
+import org.apache.log4j.Logger;
+import com.everis.boundary.CarResources;
+import com.everis.control.CarService;
+import com.everis.entity.Car;
+
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+
+
+@Path("cars")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+public class CarResources {
+
+    
+    CarService carService;
+
+    @GET
+    public Response getCars() {
+    		
+    	try {
+    		List<Car> cars = carService.getCars();
+			return Response.ok().entity(cars).build();
+		} catch (Exception e) {
+			e.printStackTrace();
+			return Response.status(Status.NOT_FOUND).build();
+		}   	
+    }
+
+    @GET
+    @Path("/{id}")
+    public Response getCarById(final @PathParam("id") int id)
+    {		
+	    try {
+			return Response.ok().entity(carService.getCar(id)).build();
+		} catch (Exception e) {
+			 return Response.status(Status.NOT_FOUND).build();
+		}
+    }
+    @POST
+    public Response createCar(final Car car) {
+    	try {
+			return Response.status(Status.CREATED).entity(carService.createCar(car)).build();
+		} catch (Exception e) {
+		    return Response.status(Status.BAD_REQUEST).build();
+		}
+	}    
+
+    @PUT
+    @Path("/{id}")
+    public Response updateCar(final @PathParam("id") int id, final Car car) {
+    	
+    	try {
+			Car newCar = this.carService.getCar(id);
+			newCar.setBrand(car.getBrand());
+			newCar.setCountry(car.getCountry());
+			newCar.setRegistration(car.getRegistration());
+			this.carService.updateCar(newCar);
```

Comment:
It is better to move the logic of updating the car to the service layer and let only API related code (http status code, response building, ect) in the resource layer.